docs/decisions/0004-downstream-effect-events.rst

```diff
@@ -0,0 +1,103 @@
+Implementation of Event Driven Architecture for Exam Downstream Effects
+=======================================================================
+
+Status
+------
+
+Draft (circa Sept 2023)
+
+Context
+-------
+For Milestone 6 of the exams IDA project, we are porting any downstream effects of exam submission and review
+from edx-proctoring to the exams IDA (aka edx-exams). Most of these downstream effects will reflect calls
```

Comment:
I generally like to steer away from using and acronym until it has been defined because not everyone is familiar with what acronyms represent.  Could you define IDA first before using it? You could even just use "service" instead.